app/experimenter/nimbus-ui/src/components/PageResults/TableVisualizationRow/index.tsx

```diff
@@ -247,7 +247,7 @@ const TableVisualizationRow: React.FC<{
   }
 
   return tableLabel === TABLE_LABEL.HIGHLIGHTS ? (
-    <div key={metricKey} className={`${className} py-2`}>
+    <div key={metricKey} className={`${className} py-2`} data-tip={tooltipText}>
```

Comment:
Ohh was this missing from previous PR?

Reply:
yes it was, I noticed it when the page actually loaded

Reply:
Actually, I checked a different experiment and realized this was doing a double tooltip. So removed this change. I'll file a separate issue because the tooltips dont show up in the case when retention is not computed